src/transformers/configuration_utils.py

```diff
@@ -362,14 +366,15 @@ def get_config_dict(cls, pretrained_model_name_or_path: str, **kwargs) -> Tuple[
         resume_download = kwargs.pop("resume_download", False)
         proxies = kwargs.pop("proxies", None)
         local_files_only = kwargs.pop("local_files_only", False)
+        revision = kwargs.pop("revision", None)
```

Comment:
Should this be "main"? Because the docstrings mentions that it defaults to that. 

```python
revision = kwargs.pop("revision", "main")
```

Reply:
It defaults to that downstream in the code, and I'd rather keep the default value in just one place.